generator/graphql-kotlin-federation/src/main/kotlin/com/expediagroup/graphql/generator/federation/extensions/CompletableFutureExtensions.kt

```diff
@@ -42,18 +42,9 @@ internal fun <T : Any?> List<CompletableFuture<out T>>.joinAll(): CompletableFut
 @Suppress("TooGenericExceptionCaught")
 internal fun <T : Any?> List<CompletableFuture<out T>>.allSettled(): CompletableFuture<List<Result<T>>> {
     val resultFutures = map { future ->
-        CompletableFuture.supplyAsync {
-            try {
-                Result.success(future.get())
-            } catch (e: Exception) {
-                Result.failure<T>(e)
-            }
-        }
+        future.thenApply { result -> Result.success(result)}
```

Comment:
this fails ktlint
```
/home/runner/work/graphql-kotlin/graphql-kotlin/generator/graphql-kotlin-federation/src/main/kotlin/com/expediagroup/graphql/generator/federation/extensions/CompletableFutureExtensions.kt:45:60 Missing spacing before "}"
```

Reply:
Ah an oversight, will get to it tomorrow!

Reply:
Done